Lib/test/clinic.test

```diff
@@ -92,7 +92,7 @@ test_objects_converter
 [clinic start generated code]*/
 
 PyDoc_STRVAR(test_objects_converter__doc__,
-"test_objects_converter($module, a, b=None, /)\n"
+"test_objects_converter($module, a, b=<unspecified>, /)\n"
```

Comment:
It would be considerable that `<unspecified>` is obvious.
PEP 570 enabled us to make text signature and document consistent.
In document, we use `[, b]` for optional argument without public default value.

Reply:
I would prefer `[, b]` if it be what the user see. But the average user will not see this, because the inspect module fails to parse both forms and the user will see just `(...)` in the help output. The purpose of this change to prevent displaying a wrong result (`b=None` when None is not acceptable). I do not see a sense of spending additional effort if the result is the same. I will make the representation using brances once I have found how to make it visible to the user.